docs/sample_config.yaml

```diff
@@ -1360,12 +1360,24 @@ saml2_config:
   #
   #grandfathered_mxid_source_attribute: upn
 
-  # Path to a file containing HTML content to serve in case an error happens
-  # when the user gets redirected from the SAML IdP back to Synapse.
-  # If no file is provided, this defaults to some minimalistic HTML telling the
-  # user that something went wrong and they should try authenticating again.
+  # Directory in which Synapse will try to find the template files below.
+  # If not set, default templates from within the Synapse package will be used.
+  #
+  # DO NOT UNCOMMENT THIS SETTING unless you want to customise the templates.
+  # If you *do* uncomment it, you will need to make sure that all the templates
+  # below are in the directory.
```

Comment:
This is a little confusing since not all the templates in the directory need to exist, just the ones listed below (I could see this being confusing).

I'm unsure if there is a better way to clarify it.

Reply:
"you will need to make sure that all the templates below are in the directory." sounds clear enough to me that we're only talking about this list of templates. fwiw we use the exact same wording for other template directories in the config.

Reply:
Works for me.